src/domains/fdr-common/03_storage_account_fdr.tf

```diff
@@ -192,25 +262,3 @@ resource "azurerm_role_assignment" "fdrconversionsa_data_contributor_role" {
     module.fdr_conversion_sa
   ]
 }
-
-# 2 - Change container Authentication method to Azure AD authentication
-resource "null_resource" "change_auth_fdr1_cached_response_blob_file" {
```

Comment:
and related apim policy ?

Reply:
This follows the same considerations on blob storage.